src/components/ProjectDetail/TaskListBoard/TaskDrawer.tsx

```diff
@@ -0,0 +1,242 @@
+                     roleCheck && setShowTextArea(true);
+                  }}
+               ></i>
+            </div>
+            <div className="taskDrawer__err">{errors.notes}</div>
+            <div className="taskDrawer__option">
+               <div className="taskDrawer__option-item">
+                  <label>Assignee</label>
+                  <select
+                     disabled={!roleCheck}
+                     onChange={(e) => {
+                        if (e.target.value) {
+                           setFormData({
+                              ...formData,
+                              assign: USERS.find(
+                                 (user) => user.id === Number(e.target.value)
+                              ),
+                           });
+                        }
+                     }}
+                  >
+                     {currentUser?.role === Role.Member &&
+                        USERS.filter((user) => user.role === Role.Member).map(
+                           (user) => (
+                              <option
+                                 key={user.id}
+                                 value={user.id}
+                              >{`${user.lastName} ${user.email}`}</option>
+                           )
+                        )}
+                     {currentUser?.role === Role.PM &&
+                        USERS.filter((user) => user.role === Role.PM).map(
+                           (user) => (
+                              <option
+                                 key={user.id}
+                                 value={user.id}
+                              >{`${user.lastName} ${user.email}`}</option>
+                           )
+                        )}
+                  </select>
+               </div>
+               <div className="taskDrawer__option-item">
+                  <label>Due date</label>
+                  <input
+                     disabled={!roleCheck}
+                     value={formData.dueDate}
+                     onChange={(e) =>
+                        setFormData({ ...formData, dueDate: e.target.value })
+                     }
+                     type="date"
+                  />
+               </div>
+               <div className="taskDrawer__option-item">
+                  <label>Status</label>
+                  <select
+                     disabled={
+                        task.status === TaskStatus.Requesting || !roleCheck
+                     }
+                     onChange={(e) => {
+                        if (e.target.value) {
+                           setFormData({
+                              ...formData,
+                              status: Number(e.target.value) as TaskStatus,
+                           });
+                        }
+                     }}
+                  >
+                     {categories.map((category) => (
+                        <option value={category}>{TaskStatus[category]}</option>
```

Comment:
mình có hàm `textFromTaskStatus` để get label nhé
```suggestion
                        <option value={category}>{textFromTaskStatus(category)}</option>
```